### DISCUSSION TITLE
kafka bridge virtual routing

### DISCUSSION THREAD

**Discussion by sreejesh-radhakrishnan-db** [strimzi/strimzi-kafka-bridge#750] [General]
Hi I have a locked down K8s cluster, so I cant even ssh. hence this Q. 

I want to use Kafka bridge and only way I can use it is expose via a Service mesh endpoint 

Do do that, I want to know what will be the URI prefix

https://<kafkabrdge:8080/<??>   I am after what will be ?? generally .

**Comment by ppatierno** [comment]
Never used a Service mesh but ...

> Do do that, I want to know what will be the URI prefix https://<kafkabrdge:8080/ I am after what will be ?? generally .

The bridge exposes multiple endpoints to deal with HTTP requests, like `/topics` and `/consumers` but followed by different subpaths depending on what you are doing. For more details check the documentation please:

https://strimzi.io/docs/bridge/latest/

**Reply by sreejesh-radhakrishnan-db** [reply]
thank you
